content/standalone-elements/standalone-elements.11tydata.js

```diff
@@ -165,5 +165,17 @@ module.exports = {
       thumbnail: "zero-md.png",
       quote: "A web component to load and display an external MD file. Uses Marked and Prism libraries under the hood.",
     },
+    {
+      url: "https://github.com/node-projects/gauge.webcomponent/",
+      title: "@node-projects/gauge.webcomponent",
```

Comment:
Consider removing npm scope, it wraps the title undesirably.

```suggestion
      title: "gauge.webcomponent",
```